.gitlab-ci.yml

```diff
@@ -43,12 +43,13 @@ Portal-Setup:
     when: always
     paths:
       - .kube/
+      - url.txt
   cache: {}
 
 Cypress-Setup:
   when: always
   stage: cypress-setup
-  image: cypress/base:10
+  image: jonsy13/cypress:buster-slim
```

Comment:
Why do we need a custom image here? Due to size constrain?

Reply:
Yes, I have added Dockerfile.Cypress for the same in this PR.

Reply:
We need to use litmuschaos image in the code. You can make use of [test-tools](https://github.com/litmuschaos/test-tools) repo for creating litmuschaos image of this Dockerfile if this image is fixed and you won't need to change it frequently. Otherwise, you can give this image from GitLab env as well like [this](https://github.com/litmuschaos/litmus-e2e/blob/7072a44c8ec2bbf9c3c6b7523bbacb17a566efb8/.gitlab-ci.yml#L16). 
It is still okay if you want to do it upcoming PR,

Reply:
Yes, I will change it accordingly in upcoming PRs.